docs/rundown.rst

```diff
@@ -162,8 +242,7 @@ conditions.
    :caption: Verifying our sample project
 
    $ ansible-sign project gpg-verify .
-   Signature validation SUCCEEDED!
-   Verification of checksum file succeeded.
+   [OK   ] GPG signature verification succeeded.
```

Comment:
Is this missing a line? There should be a line showing for both verifications if I didn't break something in ansible-sign.

Reply:
I think it might be the way mine is set up- It is giving me an error message. Perhaps we could check on this together next week? Alternatively, if you wouldn't mind giving me the expected output I can add it here. Whichever you prefer!

Reply:
Happy to work with you on figuring out where the error is coming from.